src/app/views/resources/Resources.js

```diff
@@ -0,0 +1,159 @@
+import React from "react"
+import "./Resources.scss"
+
+import { Footer } from "app/components"
+
+import ExternalLink from "../../containers/external-link/ExternalLink";
+
+import resources_tools from "assets/images/resources_tools.png";
+
+function Resources() {
+  return (
+    <div className="Resources">
+      <img className="resources-main-img" src={resources_tools} alt="Backpack and Tools" />
```

Comment:
The graphic used here seems to be a bit low in resolution. Could we get a higher resolution version?